src/shared/flow-types.js

```diff
@@ -108,6 +112,7 @@ export type BuiltAreaTaskType = {
 
 // used only by projects of type BUILDING_FOOTPRINTS (type 2)
 export type BuildingFootprintTaskType = {
+    center: ?Point,
```

Comment:
does this mean that all BuildingFootprintTasks should have a `center` attribute? For the TMS based approach you don't need it at the moment I think.

What happens if the task will not have this attribute? (Just need to know for back end side)

Reply:
The question mark in `flow` is like `Optional[number]` in python type annotations. So it can be missing and the app is still happy.